### ISSUE TITLE
please add session basef in code 

### ISSUE THREAD

**Issue by savan878** [Naeaerc20/NOTPIXEL-TGBOT#2]

**Comment by Naeaerc20** [comment]
Yes, sessions file's are already added in all versions. I'm finalizing a new version of the whole code. Index.js will be the same and used for improvements and basic tasks, index_paintworld.js will paint in templates using a specific protocol to paint on right pixels with right colors, and I'm finalizing a new file that will use my own API that I've created
